testscommon/printTxLogProcessor.go

```diff
@@ -0,0 +1,68 @@
+package testscommon
+
+import (
+	"encoding/hex"
+	"strings"
+
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/ElrondNetwork/elrond-go/core/vmcommon"
+	"github.com/ElrondNetwork/elrond-go/data"
+)
+
+var log = logger.GetOrCreate("testscommon/transactionLog")
+
+type printTxLogProcessor struct {
+}
+
+// NewPrintTxLogProcessor -
+func NewPrintTxLogProcessor() *printTxLogProcessor {
```

Comment:
why do you need tx log processor printer in testcommon ?

Reply:
My request: for debugging purposes on integration tests

Reply:
I moved this back to `transactionLog` package.